make/autoconf/jdk-version.m4

```diff
@@ -23,6 +23,26 @@
 # questions.
 #
 
+# ===========================================================================
+# (c) Copyright IBM Corp. 2019, 2019 All Rights Reserved
+# ===========================================================================
+# This code is free software; you can redistribute it and/or modify it
```

Comment:
This file just needs the copyright and not the license. See, for example, make/lib/Lib-java.base.gmk

Reply:
Aha, got it. Almost added the license to `make/lib/Lib-java.base.gmk`.